cpp/daal/src/externals/istrconv_daal_el.h

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+* Copyright 2014-2017 Intel Corporation
```

Comment:
Is it the correct year in the copyright? Is it the correct copyright?

Reply:
Yes, I'll change it.